services/ui-src/src/components/drawers/ReportDrawerDetails.tsx

```diff
@@ -42,7 +42,45 @@ export const ReportDrawerDetails = ({ entityType, drawerDetails }: Props) => {
         </Box>
       );
     case ModalDrawerEntityTypes.QUALITY_MEASURES:
-      return <Text sx={sx.detailDescription}>Quality Measures TODO</Text>;
+      return (
+        <Box sx={sx.detailBox}>
+          <Text sx={sx.detailSubtitle}>Measure Domain</Text>
+          <Text sx={sx.detailSubtext}>{drawerDetails.domain}</Text>
+          <Grid gap="normal 6" templateColumns="33% 66%">
+            <GridItem>
+              <Text sx={sx.detailSubtitle}>NQF</Text>
+              <Text sx={sx.detailSubtext}>{drawerDetails.nqfNumber}</Text>
+            </GridItem>
+            <GridItem>
+              <Text sx={sx.detailSubtitle}>Measure Reporting and Programs</Text>
+              <Text sx={sx.detailSubtext}>
+                {drawerDetails.reportingRateType}
+              </Text>
+            </GridItem>
+            <GridItem>
+              <Text sx={sx.detailSubtitle}>Measure Set</Text>
+              <Text sx={sx.detailSubtext}>{drawerDetails.set}</Text>
+            </GridItem>
+            <GridItem>
+              <Text sx={sx.detailSubtitle}>Measure Reporting Period</Text>
+              <Text sx={sx.detailSubtext}>{drawerDetails.reportingPeriod}</Text>
+            </GridItem>
+          </Grid>
+          <Text sx={sx.detailSubtitle}>Measure Description</Text>
+          <Text sx={sx.detailSubtext}>{drawerDetails.description}</Text>
+          <Heading as="h2" sx={sx.questionEyebrow}>
+            D2.VII.9a and D2.VII.9b
+          </Heading>
+          <Heading as="h3" sx={sx.questionHeader}>
+            Add plan-level measure results
+          </Heading>
+          <Text sx={sx.questionDescription}>
+            Add quality & performance measure results for specific plans. Not
+            seeing a plan? Be sure to add all plans in A: Program Information -
+            Add Plans before entering measure results.
+          </Text>
```

Comment:
Note: I'd like to talk to R&D about this design. It feels a bit odd to me because we're basically repeating the title instruction at the top of the header. It feels like this ought to be a header above the other information, and some of it is repetitive or could be omitted entirely. On most monitors we're gonna be pressed for vertical real estate, so I think it would be good to try to slim it down. See screenshot below with some notes/ideas.

![qm-drawer-design-ideas](https://user-images.githubusercontent.com/27705928/197081414-a6419593-40c0-4319-bc0f-5783ea874678.png)